BookPlayer.xcodeproj/project.pbxproj

```diff
@@ -793,7 +796,7 @@
 					"$(PROJECT_DIR)",
 				);
 				INFOPLIST_FILE = BookPlayer/Info.plist;
-				IPHONEOS_DEPLOYMENT_TARGET = 10.0;
+				IPHONEOS_DEPLOYMENT_TARGET = 10.3;
```

Comment:
you need to set the same deployment target for the target `BookPlayerTests` then travis won't complain

Reply:
Done.